script.js

```diff
@@ -38,3 +38,29 @@ document.addEventListener("DOMContentLoaded", function () {
     closeButton.style.display = "none";
   }
 });
+
+const form = document.querySelector("form");
+const errorMessage = document.createElement("p");
+errorMessage.classList.add('form-error-message')
+errorMessage.style.color = "red";
+
+function validateEmail() {
+  const emailInput = document.querySelector('input[type="email"]');
+  const formErrorInDOM = document.querySelector('.form-error-message');
+  const email = emailInput.value;
+
+  if (email !== email.toLowerCase()) {
+    errorMessage.textContent = "Please enter the email address in lowercase.";
+    formErrorInDOM ? form.appendChild(errorMessage) : '';
+    return false;
+  }
```

Comment:
- Great Job validating the email 👍 I think the error message is not showing up when the email contains Uppercases so please fix it to show the error message to improve user experience 👍